emission/core/wrapper/suggestion_sys.py

```diff
@@ -437,23 +438,32 @@ def review_start_loc_nominatim(lat, lon):
 11: "AIR_ON_HSR"
 '''
 
-def calculate_yelp_server_suggestion_singletrip_nominatim(tripid):
-    return_obj = { 'message': "Good job walking and biking! No suggestion to show.", 'method' : 'bike'}
-    start_lat, start_lon = geojson_to_lat_lon_separated(tripid["start_loc"])
-    end_lat, end_lon = geojson_to_lat_lon_separated(tripid["end_loc"])
+
+def calculate_yelp_server_suggestion_singletrip_nominatim(tripid, uuid):
+    all_users = pd.DataFrame(list(edb.get_uuid_db().find({}, {"uuid": 1, "_id": 0})))
+    user_id = all_users.iloc[all_users[all_users.uuid == uuid].index.tolist()[0]].uuid
```

Comment:
again, you have the `uuid`, but you find all the entries from the database and then find the uuid of the matching one. I think I commented on this in an earlier review but didn't insist on it being removed. You should really remove this pattern from all places in your code.
- is there any case in which `user_id != uuid`?
- you really want to avoid the use of `edb` in the code since it impedes the ability to switch to other timeseries (potentially for scalability) in the future. Please remove it from the imports so you are not tempted to use it.